tests/fips/openssl/openssl_fips_dhparam.pm

```diff
@@ -0,0 +1,34 @@
+# openssl fips test
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: testing openSSL dhparam and
+# s_client/s_server with DHE when in FIPS mode.
+# Maintainer: QE Security <none@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    assert_script_run 'openssl req -newkey rsa:2048 -nodes -keyout generatedkey.key -x509 -days 365 -out generatedcert.crt -subj "/C=DE/L=Nue/O=SUSE/CN=security.suse.de"';
+    assert_script_run 'openssl dhparam -out dhparams_2048.pem 2048';
+    clear_console;
+
+    enter_cmd 'openssl s_server -key generatedkey.key -cert generatedcert.crt -dhparam dhparams_2048.pem -cipher DHE --accept 44330';
+    assert_screen 'openssl-fips-dhparam_accept_conn';
```

Comment:
Personally, I would prefer to check the output via the console for robustness (: No need though to be changed!

Reply:
you mean using `validate_script_output` ? Because I don't think it would work here. `s_server` is a running process that must be manually terminated, and `validate_script_output` expects a process to immediately return something, right?